### PR TITLE
MNT: Re-rendered with conda-smithy 3.1.12 and pinning 2018.09.11

### PR DESCRIPTION
**Pull request by ocefpaf**
Add Python 3.7

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,25 +1,24 @@
 {% set version = "1.7.3" %}
-{% set hash = "33b878c28cf2430818e7f8c2f5705c6954fb331a109d051bad0f2e5855cbb4a4" %}
```
Comment (asmeurer) [inline recipe/meta.yaml:2]: Why this change?

Comment (ocefpaf) [reply recipe/meta.yaml:2]: That is the new standard in `conda-skeleton` and the recipes updates when I regenerate it.

Comment (asmeurer) [reply recipe/meta.yaml:2]: I always found it more useful this way so that the two things that I have to update when I release are at the top.

Comment (ocefpaf) [reply recipe/meta.yaml:2]: Feel free to revert that. Or let the bot handle the updates from now on ;-p

Comment (asmeurer) [reply recipe/meta.yaml:2]: Depends on the turnaround time for the bot. Usually for doctr I want to get releases out to conda right away.
